cmd/agent/main.go

```diff
@@ -176,6 +183,9 @@ func NewAgentRunCommand() *cobra.Command {
 	command.Flags().IntVar(&metricsPort, "metrics-port",
 		env.NumWithDefault("ARGOCD_AGENT_METRICS_PORT", cmd.ValidPort, 8181),
 		"Port the metrics server will listen on")
+	command.Flags().DurationVar(&keepAlivePingInterval, "keep-alive-ping-interval",
+		cmdutil.ParseKeepAlivePingInterval(env.StringWithDefault("KEEP_ALIVE_PING_INTERVAL", nil, "")),
```

Comment:
I think `KEEP_ALIVE_PING_INTERVAL` is not a good name for the environment variable. PTAL at how the other environment variables that we use as arguments are named, and make this one align.